src/server/json_family_test.cc

```diff
@@ -262,4 +262,231 @@ TEST_F(JsonFamilyTest, Toggle) {
   EXPECT_EQ(resp, R"([true,false,1,null,"foo",[],{}])");
 }
 
+TEST_F(JsonFamilyTest, NumIncrBy) {
+  string json = R"(
+    {"e":1.5,"a":1}
+  )";
+
+  auto resp = Run({"set", "json", json});
+  ASSERT_THAT(resp, "OK");
+
+  resp = Run({"JSON.NUMINCRBY", "json", "$.a", "1.1"});
+  ASSERT_THAT(resp, ArgType(RespExpr::STRING));
```

Comment:
you can (greatly) shorten your tests by removing `ASSERT_THAT(resp, ArgType(RespExpr::STRING));` 😄 
`EXPECT_EQ(resp, "[2.1]");` already checks that resp is of type STRING

Reply:
Indeed, I'm removing those checks.